netlink/ip.go

```diff
@@ -1,6 +1,8 @@
 // Copyright 2017 Microsoft. All rights reserved.
 // MIT License
 
+// +build linux
```

Comment:
Does this work? I thought it had to be above header?

Reply:
Certainly does in 1.11

Reply:
`...that lists the conditions under which a file should be included in the package. Constraints may appear in any kind of source file (not just Go), but they must appear near the top of the file, preceded only by blank lines and other line comments. These rules mean that in Go files a build constraint must appear before the package clause. `


https://golang.org/pkg/go/build/